test/Amadevus.RecordGenerator.TestsBase/FeaturesTestType.cs

```diff
@@ -11,9 +11,9 @@ public partial class FeatureConstructor
             // and we won't see it in reflection anyway.
             // But since we have to implement it, it's already testing
             // that the generated partial declaration exists during compilation.
-            static partial void Validate(ref string name)
+            partial void Validate()
             {
-                if (name is null) throw new System.ArgumentNullException(nameof(name));
+                if (Name is null) throw new System.ArgumentNullException("name");
```

Comment:
I'd keep this as `nameof(Name)`. Different casing is not a huge issue here IMO, but more importantly any refactoring won't "break" this.